src/main/java/sx/blah/discord/api/internal/DiscordClientImpl.java

```diff
@@ -312,15 +317,15 @@ public void login() {
 		String gateway = obtainGateway();
 		new RequestBuilder(this).setAsync(true).doAction(() -> {
 			if (shard != null) {
-				ShardImpl shardObj = new ShardImpl(this, gateway, new int[]{shard[0], shard[1]});
+				ShardImpl shardObj = new ShardImpl(this, gateway, new int[]{shard[0], shard[1]}, identifyPresence);
```

Comment:
Including the identifyPresence here might cause some inconsistencies should a user call login() with the same client object in the future. However, I am not sure if this is an actual issue or not.